python/tests/test_load_from_parquet.py

```diff
@@ -0,0 +1,434 @@
+import os
+import re
+
+import pyarrow as pa
+import pyarrow.parquet as pq
+import pytest
+
+from raphtory import Graph, PersistentGraph
+
+
+@pytest.mark.skip(reason="Prepares data for debugging purposes")
+def test_prepare_data():
+    data = {
+        "id": [1, 2, 3, 4, 5, 6],
+        "name": ["Alice", "Bob", "Carol", "Dave", "Eve", "Frank"],
+        "time": [1, 2, 3, 4, 5, 6],
+        "type": ["Person 1", "Person 2", "Person 3", "Person 4", "Person 5", "Person 6"],
+        "node_type": ["p", "p", "p", "p", "p", "p"],
+    }
+
+    table = pa.table(data)
+    pq.write_table(table, '/tmp/parquet/nodes.parquet')
+
+    data = {
+        "src": [1, 2, 3, 4, 5],
+        "dst": [2, 3, 4, 5, 6],
+        "time": [1, 2, 3, 4, 5],
+        "weight": [1.0, 2.0, 3.0, 4.0, 5.0],
+        "marbles": ["red", "blue", "green", "yellow", "purple"],
+        "marbles_const": ["red", "blue", "green", "yellow", "purple"],
+        "layers": ["layer 1", "layer 2", "layer 3", "layer 4", "layer 5"],
+    }
+
+    table = pa.table(data)
+    pq.write_table(table, '/tmp/parquet/edges.parquet')
```

Comment:
please use [tempfile](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory)
for all temporary places